frontend/.env.example

```
@@ -1,3 +1,6 @@
 VITE_API_URL=http://localhost:8000
 VITE_API_KEY=your_api_key_here
 VITE_DEBUG=false
+
+## `npx cap sync` やビルド時のみ
+CAPACITOR_UPDATER_URL=https://<r2-public-domain>/egograph/capacitor_updates/latest.json
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)